main/inc/lib/display.lib.php

```diff
@@ -1198,7 +1198,9 @@ public static function grid_js(
         if (!empty($extra_params['rowNum'])) {
             $obj->rowNum = $extra_params['rowNum'];
         }
-
+        if (function_exists('plagiarismActive') && plagiarismActive() && plagiarismTool() == TOOL_COMPILATIO) {
```

Comment:
This last comparison can be a direct comparison with the string 'compilatio'. I would prefer that. I don't see the interest of declaring a constant for that.